.github/workflows/go-semantic-release.yml

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
Looks like we didn't trigger this from dispatch, or am I misreading the go-semantic-release step in the build.yaml?

Reply:
I believe the idea is that the completion of the build workflow will trigger the semantic release workflow with the updated commits from the multimod job